boilerplate/lyft/golang_test_targets/download_tooling.sh

```diff
@@ -20,7 +20,7 @@ tools=(
   "github.com/lyft/flytestdlib/cli/pflags"
   "github.com/golangci/golangci-lint/cmd/golangci-lint"
   "github.com/alvaroloes/enumer"
-  "github.com/pseudomuto/protoc-gen-doc"
+  "github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc"
```

Comment:
you have to add this to boilerplate. When boilerplate update this will be overriden

Reply:
you will have to add it here - https://github.com/flyteorg/boilerplate/blob/master/boilerplate/flyte/golang_support_tools/tools.go

cc @evalsocket

Reply:
Done here https://github.com/flyteorg/boilerplate/pull/27

Reply:
@pmahindrakar-oss @kumare3 I also have one pending pr related to boilerplate https://github.com/flyteorg/flyteidl/pull/162 please merge it before merging it

Reply:
Awaiting the PR https://github.com/flyteorg/boilerplate/pull/27 to be merged and updated in boilerplate for flyteidl automatically